rewrite-maven/src/main/java/org/openrewrite/maven/MavenParser.java

```diff
@@ -240,11 +248,6 @@ private Builder mavenConfig(@Nullable String mavenConfig) {
                             String[] profiles = matcher.group(1).split(",");
                             activeProfiles(Arrays.stream(profiles).map(String::trim).toArray(String[]::new));
                         }
-                    } else if (line.startsWith("-D")) {
-                        Matcher matcher = Pattern.compile("-D(.*?)=(.*)").matcher(line);
-                        if (matcher.find()) {
-                            property(matcher.group(1).trim(), matcher.group(2).trim());
-                        }
```

Comment:
No need to (fragile) parse what Maven has already pulled from command line flags or maven.config file.

Reply:
Yes, agreed.